api-gateway/src/main/resources/static/scripts/bill-details/bill-details.template.html

```diff
@@ -42,6 +42,8 @@ <h5>Due Date: <strong>{{$ctrl.bills.dueDate}}</strong></h5>
             <hr>
             <br>
             <h5>Final Amount Due: {{$ctrl.bills.amount}}</h5>
+            <h5 ng-if="show">Final Amount Including Taxes: {{$ctrl.bills.taxedAmount}}</h5>
+            <button class="newBtn btn btn-primary" ng-click="show = !show"> {{show? "Hide amount including taxes" : "Show amount including taxes"}}</button>
```

Comment:
You also could have done this by crating a style class that is hidden, and then when the button is clicked the style class is then shown! Just a small change, either way works well!

Reply:
Ok, I will try to do your suggestion !